Gemfile

```diff
@@ -30,3 +30,5 @@ gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.0" if Gem.win_platform?
+
+gem "webrick", "~> 1.7"
```

Comment:
I couldn't find a reason for adding this. Why do we have it?

Reply:
Webrick is a gem required to build the website with ruby 3.0. Figured I would add it here so that any 3.0 users can simply use the `bundle install` without having to run `bundle install webrick`.